pkg/common/configs/config.go

```diff
@@ -35,7 +35,7 @@ import (
 // set of scheduler resources.
 type SchedulerConfig struct {
 	Partitions []PartitionConfig
-	Checksum   [32]byte `yaml:"-" json:"-"`
+	Checksum   [32]byte `yaml:",omitempty" json:",omitempty"`
```

Comment:
If we marshal the checksum in the config there should be no need to handle them separately in the code.
This seems to be a double up with handling in the code

Reply:
I handled them separately  because in case of config update the input will have the new configuration with the base configuration checksum.